lib/tasks/learn_to_rank.rake

```diff
@@ -137,12 +143,17 @@ namespace :learn_to_rank do
   end
 
   def fetch_latest_model_filename(bucket_name, prefix)
-    s3_objects  = Aws::S3::Bucket.new(bucket_name).objects(prefix: prefix)
-    model_files = s3_objects.map { |object| object.key.delete("#{prefix}/") }
-    model_files.max_by(&:to_i)
+    begin
+      prefix = "foo"
```

Comment:
Did you intend to include this line?

Reply:
this was me testing stuff :) removed now